src/main/java/com/google/devtools/build/lib/skylarkbuildapi/platform/PlatformConfigurationApi.java

```diff
@@ -32,15 +32,17 @@
 public interface PlatformConfigurationApi {
 
   @SkylarkCallable(name = "host_platform", structField = true, doc = "The current host platform")
-  public Label getHostPlatform();
+  Label getHostPlatform();
+
+  @SkylarkCallable(name = "platform", structField = true, doc = "The current target platform")
```

Comment:
Since this becomes a public API can we provide clearer guidance on the distinction between this and platforms?

Reply:
platforms is deprecated (updated docs to clarify).